File: homework01/HomeWork01_CSharp/Form1.cs
using System;
using System.Windows.Forms;

namespace HomeWork01_CSharp
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }


        private void button1_Click(object sender, EventArgs e)
        {
            if (txtUsername.Text == "username" && txtpassword.Text == "password")
            {
                new Form2().Show();
                this.Hide();
            } else
            {
                MessageBox.Show("The User name or password you entered is incorrect, try again!");
                txtUsername.Clear();
                txtpassword.Clear();
                txtUsername.Focus();
            }
        }

        private void label3_Click(object sender, EventArgs e)
        {
            Application.Exit();
        }

        private void label2_Click(object sender, EventArgs e)
        {
            txtUsername.Clear();
            txtpassword.Clear();
            txtUsername.Focus();
        }
    }
}

File: homework01/HomeWork01_VB.net/Form1.vb
Public Class Form1
    Private Sub Label2_Click(sender As Object, e As EventArgs)

    End Sub

    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
        If txtusername.Text = "username" And txtpassword.Text = "password" Then
            Dim form = New Form2
            form.Show()
        Else
            MessageBox.Show("The User name or password you entered is incorrect, try again!")
            txtpassword.Clear()
            txtusername.Clear()
            txtusername.Focus()
        End If
    End Sub

    Private Sub txtpassword_TextChanged(sender As Object, e As EventArgs) Handles txtpassword.TextChanged

    End Sub

    Private Sub Label3_Click(sender As Object, e As EventArgs) Handles Label3.Click
        Application.Exit()
    End Sub

    Private Sub Label2_Click_1(sender As Object, e As EventArgs) Handles Label2.Click
        txtpassword.Clear()
        txtusername.Clear()
        txtusername.Focus()
    End Sub
End Class



